src/integration/common/header.js

```diff
@@ -8,12 +8,21 @@ export default () => {
       expect(logo.parentNode.textContent).toMatchSnapshot();
     });
 
+    it("I can see the offscreen text with product's language code set to English", () => {
+      const langCode = document.querySelector(
+        'header div span span[lang="en-GB"]',
+      );
+      expect(langCode).toBeInTheDocument();
+    });
+
     it('I can see the navigation', () => {
       const navigationItemEls = document.querySelector(
         'header nav ul > li > a',
       );
 
       expect(navigationItemEls).toBeInTheDocument();
+      expect(navigationItemEls.getAttribute('href')).toBeTruthy();
+      // expect(navigationItemEls.getAttribute('text')).toBeTruthy();
```

Comment:
A snapshot comparison here would be useful to check the content is as expected. That ought to cover the Cypress test which asserts `title` against the config file. (https://github.com/bbc/simorgh/pull/6707/files#diff-6ef73c82b1834f188b8648a2ce247f32L395)

Reply:
Updated 👍